lib/active_merchant/billing/gateways/card_connect.rb

```diff
@@ -2,7 +2,10 @@ module ActiveMerchant #:nodoc:
   module Billing #:nodoc:
     class CardConnectGateway < Gateway
       self.test_url = 'https://fts.cardconnect.com:6443/cardconnect/rest/'
+      # self.test_cardsecure_url = 'https://fts.cardconnect.com:6443/cardsecure/cs'
```

Comment:
Do we want to keep this, or the url construction method that seems unused? Or is cardsecure integration still coming?

Reply:
Dammit, will axe. Thanks for the catch.